Update to poetry dependencies

Timmy-Codes opened the pull request:
Update to pyproject.toml to include PySide6 and pytest-qt dependencies
Updated poetry.lock file to reflect the above added dependencies

## Summary by Sourcery

Add PySide6 and pytest-qt dependencies.

Build:
- Add `pyside6` as a project dependency.
- Add `pytest-qt` as a development dependency.

BurdManMan committed:
Create read_dicom_file.py

Provides a function that will try to safely read a .DCM file.  If it can read the file it will return the FileDataset object in the .DCM file.  If it cannot successfully read the file it will return None.

BurdManMan committed:
Update read_dicom_file.py

Removed testing code that was accidentally left in.

Timmy-Codes committed:
Update pyproject.toml to include PySide6 and pyside-qt dependencies.

Timmy-Codes committed:
Merge branch 'UOW-Group17:dev' into dev

BurdManMan reviewed (dismissed):
Leave pytest-qt commented out.

sjswerdloff commented:
> Leave pytest-qt commented out.

or add the required packages to your ubuntu container.

sjswerdloff commented:
And if you are going to comment out pytest-qt, put in a comment explaining why it is commented out so that the team doesn't rinse and repeat *again*.

Timmy-Codes committed:
Add EGL library to Ubuntu for graphics support

Timmy-Codes committed:
Merge remote-tracking branch 'origin/dev' into dev

Timmy-Codes committed:
dev_test (#9)

Update poetry package to include PySide6 and pytest-qt dependencies
Update UnitTesting.yml to include EGL Library for Ubuntu graphics support - this was the error

Kahreiru reviewed (approved):
Tests pass, seems good to me

BurdManMan reviewed (approved):
Looks good to me, thanks for taking care of this
